torch_geometric/nn/aggr/basic.py

```diff
@@ -22,7 +22,8 @@ def forward(self, x: Tensor, index: Optional[Tensor] = None, *,
         return self.reduce(x, index, ptr, dim_size, dim, reduce='sum')
 
 
-AddAggregation = SumAggregation  # Alias
+AddAggregation = type('AddAggregation', SumAggregation.__bases__,
```

Comment:
Mmh, I see the problem. Let us instead add `"add"` support directly in `aggregation_resolver`. This should be cleaner.